python_modules/dagster/dagster/serdes/serdes.py

```diff
@@ -50,6 +50,7 @@
 class WhitelistMap(NamedTuple):
     tuples: Dict[str, TupleEntry]
     enums: Dict[str, EnumEntry]
+    names: Dict[str, Dict[str, str]]
```

Comment:
nit: the intermediate `str` keyed dict instead of two root properties or a namedtuple or something seems odd